web/src/components/spinner-button.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import '../../css/spinner-button.css';
+
+ export default class SpinnerButton extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <button className="spinnerButton spinning" disabled="disabled">Submit</button>
```

Comment:
I think `Submitting...`  would be a better label for the button in this case, to better indicate what is going on. What do you think?

Reply:
Yes, more explanatory than spinning ;

Reply:
Oh, I see know what you meant